frappe_graphql/utils/resolver.py

```diff
@@ -131,6 +132,22 @@ def save_doc_resolver(obj: Any, info: GraphQLResolveInfo, **kwargs):
         "doc": doc.as_dict()
     }
 
+def delete_doc_resolver(obj: Any, info: GraphQLResolveInfo, **kwargs):
+    doctype = kwargs["doctype"]
+    name = kwargs["name"]
+    success  = False
+    try:
+        doc = frappe.get_doc(doctype, name)
+        doc.delete()
+        success = True
+    except Exception as e:
```

Comment:
@e-lobo shouldn't we throw the error as it is ?

Reply:
@faztp12 @assemmarwan there will be `_server_messages` for perm or link validation errors.. but i can `frappe.msgprint(frappe.get_traceback())` if needed, should i?

Reply:
i think _server_messages shouldnt be relied on if we are to follow the graphql spec.

Reply:
@faztp12 what do you suggest?

Reply:
@e-lobo simply throw the exception as it is, no try catch. what do you think ?

Reply:
@faztp12 done 👍